test/e2e_node/e2e_service.go

```diff
@@ -190,7 +190,9 @@ func (es *e2eService) startEtcd() (*killCmd, error) {
 		}
 		etcdPath = defaultEtcdPath
 	}
-	cmd := exec.Command(etcdPath)
```

Comment:
We should change the lookpath bit above to try `etcd2` as the binary name and switch to `etcd` if it doesn't find it there. CoreOS ships both etcd and etcd2 currently, so this is using an old and not recommended version of etcd in that case, and it's possible another distro might follow this lead.